rosetta-lang/src/main/java/com/regnosys/rosetta/types/SubtypeRelation.java

```diff
@@ -57,11 +69,11 @@ public boolean isSubtypeOf(RType t1, RType t2, boolean treatChoiceTypesAsDataTyp
 		} else if (t1 instanceof RChoiceType) {
 			RType t1_ = t1;
 			RType t2_ = t2;
-			return ((RChoiceType)t1).getOwnOptions().stream().allMatch(t -> safeIsSubtypeOf(t.getType(), t2_, false, t1_, visited));
+			return ((RChoiceType)t1).getOwnOptions().stream().allMatch(t -> safeIsSubtypeOf(t.getType().getRType(), t2_, false, t1_, visited));
```

Comment:
These two should not remove metadata.

Reply:
Have something for this locally, lets review on Monda

Reply:
Reviewed and decided not applicable as meta is not considered